cypress/e2e/specialFeatures/liteSiteWeight/helpers/getTotalPageSize.js

```diff
@@ -11,6 +11,12 @@ export default requests => {
         totalSize += contentLength;
       } else {
         getPageSizeInKB(url).then(size => {
+          cy.task('table', [
+            {
+              Request: url,
+              'Request Size (KB)': size,
+            },
+          ]);
```

Comment:
I wonder if we should put the URLs + sizes into another array, and then log it once? Because we're getting multiple tables in the output now:

![Screenshot 2025-04-04 at 10 57 19](https://github.com/user-attachments/assets/a445b60e-0534-4608-9fdc-ad56d68ba3e6)